securedrop/source_templates/lookup.html

```diff
@@ -18,10 +18,14 @@
 </div>
 
 <h1 class="headline">{{ gettext('Submit Files or Messages') }}</h1>
-<p class="explanation">{{ gettext('If you are already familiar with GPG, you can optionally encrypt your files and messages with our <a href="{url}" class="text-link">public key</a> before submission. Files are encrypted as they are received by SecureDrop.').format(url=url_for('info.download_journalist_pubkey')) }}
-{{ gettext('<a href="{url}" class="text-link">Learn more</a>.').format(url=url_for('info.why_download_journalist_pubkey')) }}</p>
 
-<p class="explanation">{{ gettext('You can submit any kind of file, a message, or both.') }}</p>
+<p class="explanation"><b>1.</b> {{ gettext('Before you begin, <a href="{url}" class="text-link">turn off cross-site request sanitization in NoScript</a>. This is <strong>required</strong> for uploads to work.').format(url=url_for('info.disable_noscript_xss'))}}</p>
+
```

Comment:
Per the [final messaging](https://docs.google.com/document/d/1HNrRaOrRJfUX61DDNRpQcc9UKs6y80Y9DWUf3PxypU4/edit?ts=5c6768ff), instead of linking the "turn off" text, there should be a "Show me how" link at the end. (The UX reasoning is that a "turn off" link could cause inexperienced users to think that clicking the link will immediately change their settings.)

Reply:
Fixed.

Reply:
LGTM, thanks!